src/Joblify.Core/Offers/OfferService.cs

```diff
@@ -19,13 +23,33 @@ public OfferService(IMapper mapper, IUnitOfWork unitOfWork)
         public async Task<OfferDto> AddOfferAsync(OfferDto offerDto)
         {
             var offer = _mapper.Map<Offer>(offerDto);
-
             await _unitOfWork.OfferRepository.AddAsync(offer);
             await _unitOfWork.CommitAsync();
 
             var created = _mapper.Map<OfferDto>(offer);
```

Comment:
I know it was here before, but do we need this 2nd mapping?
We basically map dto to offer, then add offer and map offer back to dto - it's still the same object that was passed at the beginning :)

Reply:
It was my code, fixed it on my branch "search offers"